Phonebook.Frontend/src/app/modules/table/dialogs/table-settings-dialog/table-settings.dialog.html

```diff
@@ -61,3 +62,6 @@ <h2 i18n="TableSettingsDialog|Title of displayed table columns@@TableSettingsDia
     </div>
   </div>
 </mat-dialog-content>
+<mat-dialog-actions style="justify-content: flex-end;">
+  <button mat-stroked-button mat-icon-button color="primary" mat-dialog-close><mat-icon>close</mat-icon></button>
```

Comment:
```suggestion
  <button mat-stroked-button color="primary" mat-dialog-close>Close</button>
```

I would change it like this because it will be similar to our other dialogs

Reply:
A closing `X` could also be added at the top if you want. This would duplicate the action but adds visibility on smaller devices.

Reply:
Like this: 
![image](https://user-images.githubusercontent.com/13590797/63353767-8dd16600-c363-11e9-9547-7c20863b1cb0.png)

Reply:
Stroked buttons are used nowhere in the application.

Reply:
Okay @J-C-O, we discussed this and it would be cool if on the right top there is this little cross. like on the small cards. Please change your X-Button to "Close" as a normal `mat-button` with black text color. It will also be cool when you move the Reset-Button next to the close button on the bottom.